### PR TITLE
[US12 Task42] FE add a field for Project slug

### PR DESCRIPTION
**Pull request by spark290**
US12 Task42 Front-end add a field for project slug

### CODE REVIEW

**Comment by AdvaitChirmule** [comment]
Hi, the packages.json is missing the required dependencies for bootstrap and react_bootstrap

**Comment by spark290** [comment]
> Hi, the packages.json is missing the required dependencies for bootstrap and react_bootstrap

Added packages.json file

**Comment by AdvaitChirmule** [comment]
It is fixed now

File: taigaProject/taigaApi/controller/auth_controller.py [hunk 1]
```diff
@@ -12,10 +11,3 @@ def auth(auth: AuthRequest):
     token = authenticate(auth.username, auth.password)
     if token == None:
         raise HTTPException(status_code=401, detail="Invalid Login Credentials")
-
```
Comment (shahih25) [inline taigaProject/taigaApi/controller/auth_controller.py]: Please revert this change.

**Comment by shahih25** [comment]
Please remove .idea folder which was pushed. Also we shouldn't be pushing the pycache changes.

**Comment by shahih25** [comment]
Can you attach the screenshot of the screens?

File: react-ui/README.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+## Download Node.js
```
Comment (shahih25) [inline react-ui/README.md]: Delete this README as we already have outside of this folder.

File: react-ui/public/index.html [hunk 3]
```diff
@@ -24,6 +24,9 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css"
+      integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous"
+/>
```
Comment (shahih25) [inline react-ui/public/index.html:29]: nit: format

File: react-ui/src/ProjectSlug.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import Button from 'react-bootstrap/Button'
```
Comment (shahih25) [inline react-ui/src/ProjectSlug.js]: nit: _import { Button, Form } from "react-bootstrap";_

react-bootstrap imports could be combined into one import.

File: react-ui/public/index.html (hunk 3 above)
Comment (shahih25) [reply react-ui/public/index.html:29]: You can remove this line as it already present on US12 branch.

https://github.com/ser516asu/SER516-Team-Cleveland/blob/cc1c8b90f4d85a11b4c72f6476e3673eb9b25aa7/react-ui/public/index.html#L17

File: taigaProject/taigaApi/authenticate.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import os
```
Comment (shahih25) [inline taigaProject/taigaApi/authenticate.py]: You can delete this file as it is moved to **auth/authenticate.py**

https://github.com/ser516asu/SER516-Team-Cleveland/blob/cc1c8b90f4d85a11b4c72f6476e3673eb9b25aa7/taigaProject/taigaApi/auth/authenticate.py
